Dockerfile

```diff
@@ -1,35 +1,87 @@
-FROM r-base:4.0.3
-
-MAINTAINER https://github.com/pacificclimate/chickadee
-LABEL Description="chickadee WPS" Vendor="Birdhouse" Version="0.1.0"
+FROM rocker/r-ver:4.0.3 AS builder
 
 ENV PIP_INDEX_URL="https://pypi.pacificclimate.org/simple/"
 
-WORKDIR /code
-
 COPY requirements.txt r_requirements.txt install_pkgs.R ./
 
 # Install python and R packages
+RUN apt-get update && \
+  apt-get install -y --no-install-recommends \
+    build-essential \
+    gcc \
+    libpq-dev \
+    python3.8 \
+    python3-pip \
+    python3-setuptools \
+    python3-dev \
+    libssl-dev \
+    libxml2-dev \
+    libudunits2-dev \
+    libnetcdf-dev \
+    libgit2-dev && \
+    Rscript install_pkgs.R r_requirements.txt && \
+    pip3 install --upgrade pip && \
+    pip3 install -r requirements.txt --ignore-installed --user && \
+    pip3 install gunicorn --user
+
+FROM rocker/r-ver:4.0.3 AS prod
+MAINTAINER https://github.com/pacificclimate/chickadee
+LABEL Description="chickadee WPS" Vendor="Birdhouse" Version="0.1.0"
```

Comment:
We can probably change the `vendor` and `version` attributes to be more accurate.